src/algebra/group_ring_action.lean

```diff
@@ -40,7 +40,7 @@ class mul_semiring_action (M : Type u) (R : Type v) [monoid M] [semiring R]
 
 section semiring
 
-variables (M G : Type u) [monoid M] [group G]
+variables (M G K : Type u) [monoid M] [monoid K] [group G]
```

Comment:
M and K should be in different universes

Reply:
... and you can achieve that with `(M G K : Type*)`

Reply:
OK, thank you. Done